src/index.ts

```diff
@@ -41,10 +51,82 @@ export {SearchResultsItem} from "./molecules/churchLocator/searchResultsItem/Sea
+export {Figure} from "./molecules/media/figure/Figure";
+
+// ----- Molecules - navigation         -----
+export {Breadcrumbs} from "./molecules/navigation/breadcrumbs/Breadcrumbs";
+export {DrawerLinks} from "./molecules/navigation/drawerLinks/DrawerLinks";
+export {DrawerNavigation} from "./molecules/navigation/drawerNavigation/DrawerNavigation";
+export {FooterPrimaryNavigation} from "./molecules/navigation/footerPrimaryNavigation/FooterPrimaryNavigation";
+export {FooterSecondaryNavigation} from "./molecules/navigation/footerSecondaryNavigation/FooterSecondaryNavigation";
+export {Pagination} from "./molecules/navigation/pagination/Pagination";
+export {PrimaryNavigation} from "./molecules/navigation/primaryNavigation/PrimaryNavigation";
+export {SecondaryNavigation} from "./molecules/navigation/secondaryNavigation/SecondaryNavigation";
+
+// ----- Molecules - store         -----
+export {CartAdd} from "./molecules/store/cartAdd/CartAdd";
+export {CartItem} from "./molecules/store/cartItem/CartItem";
+export {CartItemSummary} from "./molecules/store/cartItemSummary/CartItemSummary";
+export {CheckoutBilling} from "./molecules/store/checkoutBilling/CheckoutBilling";
+export {CheckoutInfo} from "./molecules/store/checkoutInfo/CheckoutInfo";
+export {CheckoutPlace} from "./molecules/store/checkoutPlace/CheckoutPlace";
+export {CheckoutProducts} from "./molecules/store/checkoutProducts/CheckoutProducts";
+export {Notification} from "./molecules/store/notification/Notification";
+export {Product} from "./molecules/store/product/Product";
+export {ProductDetail} from "./molecules/store/productDetail/ProductDetail";
+export {Products} from "./molecules/store/products/Products";
+export {Totals} from "./molecules/store/totals/Totals";
+
+// ----- Molecules - text         -----
+export {Byline} from "./molecules/text/byline/Byline";
+export {ExtendedQuote} from "./molecules/text/extendeQuote/ExtendedQuote";
+export {Highlight} from "./molecules/text/highlight/Highligh";
+export {Pullquote} from "./molecules/text/pullquote/Pullquote";
+
+// ----- Organisms - asides         -----
+export {Aside} from "./organisms/asides/aside/Aside";
+export {BlockWrap} from "./organisms/asides/aside/BlockWrap";
+export {RelatedPosts} from "./organisms/asides/relatedPosts/RelatedPosts";
+export {Sabbath} from "./organisms/asides/sabbath/Sabbath";
+
+// ----- Organisms - content         -----
+export {ArticleContent} from "./organisms/content/articleContent/ArticleContent";
+export {ArticleContentShort} from "./organisms/content/articleContentShort/ArticleContentShort";
+export {BodyContent} from "./organisms/content/bodyContent/BodyContent";
+export {ListContent} from "./organisms/content/listContent/ListContent";
+export {PageContent} from "./organisms/content/pageContent/PageContent";
+export {SearchContent} from "./organisms/content/searchResult/SearchContent";
+export {SearchResult} from "./organisms/content/searchResult/SearchResult";
```

Comment:
Wrong path. Please check this path `"./organisms/content/searchResult/SearchResult";`